packages/web-media-box/src/hls-parser/types/parsedPlaylist.d.ts

```diff
@@ -89,26 +89,33 @@ export interface Resolution {
 export type CpcRecord = Record<string, string[]>;
 export type AllowedCpc = Array<CpcRecord>;
 
-export interface VariantStream {
+export interface BaseStreamInf {
   uri: string;
   bandwidth: number;
   averageBandwidth?: number;
   score?: number;
   codecs?: string[];
   supplementalCodecs?: string[];
   resolution?: Resolution;
-  frameRate?: number;
   hdcpLevel?: 'TYPE-0' | 'TYPE-1' | 'NONE';
   allowedCpc?: AllowedCpc;
   videoRange?: 'SDR' | 'HLG' | 'PQ';
   stableVariantId?: string;
-  audio?: string;
   video?: string;
+  pathwayId?: string;
+}
+
+// VariantStream properties that are not in BaseStreamInf
+export interface VariantStreamSpecific {
+  frameRate?: number;
+  audio?: string;
   subtitles?: string;
   closedCaptions?: string;
-  pathwayId?: string;
 }
 
+export interface VariantStream extends BaseStreamInf, VariantStreamSpecific {}
+export interface iFramePlaylist extends BaseStreamInf {}
```

Comment:
`IFramePlaylist`

Reply:
Yup, thanks. I've typed "iFrame" so many times in my life I did it by habit.